packages/grafana-ui/src/components/Table/types.ts

```diff
@@ -26,11 +26,11 @@ export interface TableRow {
 }
 
 export type TableFilterActionCallback = (key: string, value: string) => void;
-export type TableColumnResizeActionCallback = (fieldIndex: number, width: number) => void;
+export type TableColumnResizeActionCallback = (fieldDisplayName: string, width: number) => void;
 export type TableSortByActionCallback = (state: TableSortByFieldState[]) => void;
 
 export interface TableSortByFieldState {
-  fieldIndex: number;
+  displayName: string;
```

Comment:
Was thinking if actually Field having a guid isn't a good idea long term. For now displayName is OK but taking field options and overrides that will require us to update the sortBy property when they change doesn't sound optimal.

Reply:
how would we assign a guid?? needs to be matched to a query result. field display name is all we have.